lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwiftTypeRef.h

```diff
@@ -393,7 +393,22 @@ class TypeSystemSwiftTypeRef : public TypeSystemSwift {
   std::string
   AdjustTypeForOriginallyDefinedInModule(llvm::StringRef mangled_typename);
 
+  /// Desugar to this node and if it is a type alias resolve it by
+  /// looking up its type in the debug info.
+  swift::Demangle::NodePointer
+  Canonicalize(swift::Demangle::Demangler &dem,
```

Comment:
Why were these functions moved to be public? I don't see you using them anywhere outside TypeSystemSwiftTypeRef

Reply:
Fixed.